### PR TITLE
Engineering ERT members get power tools

### PR DESCRIPTION
**Pull request by NotRanged**
## About The Pull Request

Engi ERT now get power tools, upgraded RCDs, and a rapid pipe dispenser. Why would NT send in their 'elite' emergency squad with worse gear than the station gets?

## Why It's Good For The Game

Gives engi ERT the proper gear they deserve.

## Changelog
:cl:
tweak: Engi ERT now gets power tools, upgraded RCDs, and pipe dispensers.
/:cl:

### CODE REVIEW

File: code/modules/clothing/outfits/ert.dm [hunk 1]
```diff
@@ -131,12 +131,13 @@
 	suit_store = /obj/item/gun/energy/e_gun
 	glasses =  /obj/item/clothing/glasses/meson/engine
 	back = /obj/item/storage/backpack/ert/engineer
-	belt = /obj/item/storage/belt/utility/full
+	belt = /obj/item/storage/belt/utility/full/powertools/
```
Comment (Fikou) [inline code/modules/clothing/outfits/ert.dm:134]:
```suggestion
	belt = /obj/item/storage/belt/utility/full/powertools
```
i will hit you with a belt

Comment (NotRanged) [reply code/modules/clothing/outfits/ert.dm:134]: pain

**Comment by optimumtact** [comment]
Power creep in action

**Comment by LemonInTheDark** [comment]
`Why would NT send in their 'elite' emergency squad with worse gear than the station gets?`
This is depressing

**Pull request merged**